jwst/residual_fringe/residual_fringe_step.py

```diff
@@ -31,6 +33,32 @@ def process(self, input):
         self.transmission_level = 80  # sets the transmission level to use in the regions file
         # 80% is what other steps use.
 
+        # set up the dictionary to ignore wavelength regions in the residual fringe correction
+        ignore_regions = {}
+        ignore_regions['num'] = 0
+        ignore_regions['min'] = []
+        ignore_regions['max'] = []
+        if self.ignore_region_min is not None:
+            for region in self.ignore_region_min:
+                ignore_regions['min'].append(float(region))
+
+        min_num = len(ignore_regions['min'])
+
+        if self.ignore_region_max is not None:
+            for region in self.ignore_region_max:
+                ignore_regions['max'].append(float(region))
+        max_num = len(ignore_regions['max'])
+
+        if max_num != min_num:
+            self.log.error("Number of minimum and maximum wavelengths to ignore are not the same")
+
+        ignore_regions['num'] = min_num
+
+        if min_num > 0:
+            self.log.info('Ignoring {} wavelength regions'.format(min_num))
+            raise IncorrectIgnoreRegions("Number of ignore_region_min does not match ignore_region_max")
```

Comment:
Shouldn't this exception occur after line 53, where a mismatch between min_num/max_num is detected? This block of code is part of the normal operations.

Reply:
Yes. My stupid mistake